pkg/virtctl/vmexport/vmexport_test.go

```diff
@@ -243,7 +243,7 @@ var _ = Describe("vmexport", func() {
 			Expect(err.Error()).Should(ContainSubstring(expectedError))
 		})
 
-		It("VirtualMachineExport download fails if the server returns a bad status", func() {
```

Comment:
Maybe there should be a test where the download succeeds in the second or third attempt, to be confident that the retrial actually works.

Reply:
Good idea, done